### PR TITLE
add(x11): new package eww

### LINKED ISSUE #11940: [Package]: eww
### Why do you want to add this package to Termux?

I want this for this rice https://github.com/gh0stzk/dotfiles

### Home page URL

https://github.com/elkowar/eww

### Source code URL

https://github.com/elkowar/eww/archive/refs/tags/v0.4.0.tar.gz

### Packaging policy acknowledgement

- [X] The requested package complies with [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

Just too lazy to build it

### PR DESCRIPTION
**Pull request by T-Dynamos**
Closes #11940

### CODE REVIEW

**Comment by T-Dynamos** [comment]
Please merge this also, I need it!

**Comment by OrdinaryEnder** [comment]
Ewwwwwwww

File: x11-packages/eww/fix_tmp_hardcoded_path.patch [hunk 1]
```diff
@@ -0,0 +1,23 @@
+diff '--color=auto' -uNr eww/crates/eww/src/paths.rs eww.mod/crates/eww/src/paths.rs
+--- eww/crates/eww/src/paths.rs	2022-09-20 10:59:18.645199645 +0000
++++ eww.mod/crates/eww/src/paths.rs	2022-09-20 10:57:51.443494861 +0000
+@@ -35,7 +35,7 @@
+ 
+         let ipc_socket_file = std::env::var("XDG_RUNTIME_DIR")
+             .map(std::path::PathBuf::from)
+-            .unwrap_or_else(|_| std::path::PathBuf::from("/tmp"))
++            .unwrap_or_else(|_| std::path::PathBuf::from("/data/data/com.termux/files/usr/tmp"))
```
Comment (Grimler91) [inline x11-packages/eww/fix_tmp_hardcoded_path.patch:9]: Please use @TERMUX_PREFIX@ in patches

**Review by Grimler91 — CHANGES_REQUESTED** [review]

**Comment by T-Dynamos** [comment]
@Grimler91

**Comment by Grimler91** [comment]
Thanks!

**Pull request merged**